src/main/java/core/basesyntax/dao/impl/CommentDaoImpl.java

```diff
@@ -12,21 +15,64 @@ public CommentDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Comment create(Comment entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.save(entity);
+            transaction.commit();
+            return entity;
+        } catch (Exception e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Can't create comment " + entity, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
     }
 
     @Override
     public Comment get(Long id) {
-        return null;
+        try (Session session = factory.openSession()) {
+            return session.get(Comment.class, id);
+        } catch (Exception e) {
+            throw new RuntimeException("Comment with id = " + id + "isn't exist", e);
+        }
     }
 
     @Override
     public List<Comment> getAll() {
-        return null;
+        Query<Comment> query;
+        try (Session session = factory.openSession()) {
+            query = session.createQuery("SELECT e FROM Comment e", Comment.class);
```

Comment:
```suggestion
            query = session.createQuery("FROM Comment", Comment.class);
```
let's simplify all `getAll()` queries like this